### PR TITLE
Blog post for Bazel 0.26.0

### PR DESCRIPTION
**Pull request by aehlig**

### CODE REVIEW

File: _posts/2019-05-28-bazel-0.26.0.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+## Android
+
+- Android resource conflicts will no longer be reported between a strong attr
+  resource and a weak attr resource, if the weak attr does not have format
+  specified.
+
+## C++
+
+- [Allow debugging C++ features logic](https://github.com/bazelbuild/bazel/commit/53156db1e30fe7197889c3cbc8d1321ff9b85143
```
Comment (laurentlb) [inline _posts/2019-05-28-bazel-0.26.0.md]: missing `)`

File: _posts/2019-05-28-bazel-0.26.0.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  [The Bazel Opt-in Preview is Here!](https://www.youtube.com/watch?v=J1lnp-nU4wM)
+
+- The latest release of
+  [rules_haskell](https://github.com/tweag/rules_haskell/releases/tag/v0.9)
+  includes many new improvements, including an initial Windows support.
+
+- The latest
+  [Buildifier release](https://github.com/bazelbuild/buildtools/releases/tag/0.25.1)
+  includes Windows binaries, as well as a JSON output for diagnostics.
+
```
Comment (laurentlb) [inline _posts/2019-05-28-bazel-0.26.0.md]: Missing a note about VS Code:

- A new release of the [Visual Studio Code plugin for Bazel](https://marketplace.visualstudio.com/items?itemName=BazelBuild.vscode-bazel) was released.

File: _posts/2019-05-28-bazel-0.26.0.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+- [Allow debugging C++ features logic](https://github.com/bazelbuild/bazel/commit/53156db1e30fe7197889c3cbc8d1321ff9b85143
+
+- Add new options [`--cs_fdo_instrument`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--cs_fdo_instrument)
+  and [`--cs_fdo_profile`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--cs_fdo_profile)
+  to support LLVM's context-sensitive FDO (CSFDO).
+
+- [Bazel C++ compile/link Starlark API](https://github.com/bazelbuild/bazel/issues/4570).
+  Can be used with experimental flag `--experimental_cc_skylark_api_enabled_packages`.
+
+- `cc_toolchain.static_runtime_lib` and `cc_toolchain.dynamic_runtime_lib` are
```
Comment (laurentlb) [inline _posts/2019-05-28-bazel-0.26.0.md]: These should be links

Comment (aehlig) [reply _posts/2019-05-28-bazel-0.26.0.md]: Links to what? I couldn't find any documentation for those.

Comment (aehlig) [reply _posts/2019-05-28-bazel-0.26.0.md]: cc @hlopko

Comment (aehlig) [reply _posts/2019-05-28-bazel-0.26.0.md]: added links. Thank @hlopko for looking them up.

**Comment by laurentlb** [comment]
Markdown is broken. Please fix `static_runtime_lib` and linkify the links.

**Pull request merged**